translations.json

```diff
@@ -293,6 +293,55 @@
         "loading":"Lataa...",
         "function_unavailable_in_inline_mode": "Tämä toiminto ei ole käytettävissä sisäisessä tilassa"
     },
+    "he": {
+        "help_description": "הצג הודעת עזרה",
+        "reset_description": "אתחל את השיחה. ניתן להעביר הוראות ברמה גבוהה (למשל, /reset אתה עוזר מועיל)",
+        "image_description": "צור תמונה מהפרומפט (למשל, /image חתול)",
+        "tts_description": "צור דיבור מטקסט (למשל, /tts הבית שלי)",
+        "stats_description": "קבל את סטטיסטיקות השימוש הנוכחיות שלך",
+        "resend_description": "שלח מחדש את ההודעה האחרונה",
+        "chat_description": "שוחח עם הבוט!",
+        "disallowed": "מצטערים, אינך מורשה להשתמש בבוט זה. תוכל לבדוק את הקוד המקור ב https://github.com/n3d1117/chatgpt-telegram-bot",
+        "budget_limit": "מצטערים, הגעת למגבלת השימוש שלך.",
+        "help_text": ["אני בוט של ChatGPT, דבר איתי!", "שלח לי הודעה קולית או קובץ ואני אתרגם אותו בשבילך", "קוד פתוח ב https://github.com/n3d1117/chatgpt-telegram-bot"],
+        "stats_conversation": ["שיחה נוכחית", "הודעות צ'אט בהיסטוריה", "אסימוני צ'אט בהיסטוריה"],
+        "usage_today": "שימוש היום",
+        "usage_month": "שימוש החודש",
+        "stats_tokens": "אסימונים",
+        "stats_images": "תמונות שנוצרו",
+        "stats_vision": "אסימוני תמונה שפורשו",
+        "stats_tts": "תווים שהומרו לדיבור",
+        "stats_transcribe": ["דקות ו", "שניות שהוקלטו"],
+        "stats_total": "💰 לסך כל של $",
+        "stats_budget": "התקציב הנותר שלך",
+        "monthly": " לחודש זה",
+        "daily": " להיום",
+        "all-time": "",
+        "stats_openai": "החשבון שלך ב-OpenAI חויב החודש ב-$",
+        "resend_failed": "אין לך מה לשלוח מחדש",
+        "reset_done": "בוצע!",
+        "image_no_prompt": "נא לספק פרומפט! (למשל, /image חתול)",
+        "image_fail": "נכשל ביצירת התמונה",
+        "vision_fail": "נכשל בפרשנות התמונה",
+        "tts_no_prompt": "נא לספק טקסט! (למשל, /tts הבית שלי)",
+        "tts_fail": "נכשל ביצירת הדיבור",
+        "media_download_fail": ["נכשל בהורדת קובץ השמע", "ודא שהקובץ אינו גדול מדי. (מקסימום 20MB)"],
+        "media_type_fail": "סוג קובץ לא נתמך",
+        "transcript": "תמליל",
+        "answer": "תשובה",
+        "transcribe_fail": "נכשל בתרגום הטקסט",
+        "chat_fail": "נכשל בקבלת תגובה",
+        "prompt": "פרומפט",
+        "completion": "השלמה",
+        "openai_rate_limit": "חריגה מהגבלת השימוש של OpenAI",
+        "openai_invalid": "בקשה לא חוקית של OpenAI",
+        "error": "אירעה שגיאה",
+        "try_again": "נא לנסות שוב מאוחר יותר",
+        "answer_with_chatgpt": "ענה באמצעות ChatGPT",
+        "ask_chatgpt": "שאל את ChatGPT",
+        "loading": "טוען...",
+        "function_unavailable_in_inline_mode": "הפונקציה לא זמינה במצב inline"
+      }
```

Comment:
@ShadovvBeast @n3d1117 Guys, we lost a comma here

Reply:
yup, already fixed in master :)